doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,30 +94,159 @@ between tenants
+| TCP      | 30000-32767 | NodePort Services     |
+
+### 5.4.2 Controlling access to the Kubernetes API
+The Kubernetes platform is controlled using API requests and as such is the
+first line of defence against attackers. Controlling who has access and what
+actions they are allowed to perform is the primary concern
+    
+### 5.4.3 Use Transport Layer Security
+Communication in the cluster between services should be handled using TLS,
+encrypting all traffic by default. This, however, is often overlooked with the
+thought being that the cluster is secure and there is no need to provide
+encryption in transit within the cluster.
+
+Advances in network technology, such as the service mesh, have led to the
+creation of products like LinkerD and Istio which can enable TLS by default
+while providing extra telemetry information on transactions between services.
+
+Kubernetes expects that all API communication in the cluster is encrypted by
+default with TLS, and the majority of installation methods will allow the
+necessary certificates to be created and distributed to the cluster components.
+Note that some components and installation methods may enable local ports over
+HTTP and administrators should familiarize themselves with the settings of each
+component to identify potentially unsecured traffic.
+
+### 5.4.4 API Authentication, API Authorization
 Secure all connections to a Kubernetes Cluster. Adopt the following security
 authentication mechanisms:
- - Configure user roles and access levels to provide segregation of duties (RBAC)
- - Use multi-factor authentication for all administrative access
- - Use token-based or certificate-based service and session authentication mechanisms
- - Integrated with existing identity management platforms e.g SAML, AD, etc. for
- access control
+* Configure user roles and access levels to provide segregation of duties (RBAC)
+* Use multi-factor authentication for all administrative access
+* Use token-based or certificate-based service and session authentication
+  mechanisms
+* Integrated with existing identity management platforms e.g SAML, AD, etc. for
+  access control
+
+### 5.4.5 Restrict access to etcd and encrypt contents within etcd
+etcd is a critical Kubernetes component which stores information on state and
+secrets, and it should be protected differently from the rest of your cluster.
+Write access to the API server's etcd is equivalent to gaining root on the
+entire cluster, and even read access can be used to escalate privileges fairly
+easily.
+
+The Kubernetes scheduler will search etcd for pod definitions that do not have a
+node. It then sends the pods it finds to an available kubelet for scheduling.
+Validation for submitted pods is performed by the API server before it writes
+them to etcd, so malicious users writing directly to etcd can bypass many
+security mechanisms - e.g. PodSecurityPolicies.
+
+Administrators should always use strong credentials from the API servers to
```

Comment:
```suggestion
Administrators should always use strong credentials for the API servers to
```

Reply:
Update. Please review

Reply:
@ragashreeshekar Sorry I don't see a change. The way I read it this sentence seems to suggest that the API server is the source of the "strong credentials". Firstly the sentene needs to be broken up. Secondly, I think in the first part you are trying to say that for all platform components, such as API server, etcd server, etc., the administrator must utilize strong credentials.